meta(issues): Add issue templates (#645)
getsentry/onpremise: .github/ISSUE_TEMPLATE/feature_request.md

---
name: 🧠  Feature request
about: Suggest an idea for this project

---

<!--

Note that the Sentry team has finite resources and priorities that are not

> always visible on GitHub. If your issue doesn't align with our priorities it's

Reviewer: ```suggestion
always visible on GitHub. If your issue doesn't align with our priorities, it's
```

unlikely it will be worked on. If we're interested in a particular feature however,

> we'll follow up and ask you to submit an RFC to talk about it in more detail.

Reviewer: ```suggestion
we'll follow up and ask you to submit an RFC to discuss it in more detail.
```

-->

## Summary

One paragraph description of the feature.

## Motivation

Why should this be worked on? What problems or use cases does it solve or
improve?

## Additional Context

> Any other context or screenshots or API request payload/responses that you

Reviewer: ```suggestion
Any other context or screenshots or API request payload/responses that
```
